deck/tests/test_game.py

```diff
@@ -41,7 +40,8 @@ def test_dealer_discard_pile_update() -> None:
     d = Dealer()
     p1 = Player()
     p2 = Player()
-    p1, p2 = d.deals([p1, p2])
+    lp = {'richie':p1, 'libbie':p2}
```

Comment:
this is a really awkward API. How come we don't just call `player.name`?

Reply:
I'm pretty sure that this is because we need the names as keys to iterate over the players. 

Maybe I could just create a `Players` object to encapsulate this?

Reply:
Yeah, fixed this